Product/Production/Services/AuditRepositoryCore/src/main/java/gov/hhs/fha/nhinc/audit/transform/AuditTransforms.java

```diff
@@ -112,7 +116,7 @@ public final LogEventRequestType transformRequestToAuditMsg(T request, Assertion
         return buildLogEventRequestType(auditMsg, direction, getMessageCommunityId(assertion, target, isRequesting),
             serviceName, assertion, auditMsg.getEventIdentification().getEventID().getDisplayName(),
             auditMsg.getEventIdentification().getEventOutcomeIndicator(),
-            auditMsg.getEventIdentification().getEventDateTime(), getUserId(auditMsg.getActiveParticipant()));
+            auditMsg.getEventIdentification().getEventDateTime(), getRequestorUserId(auditMsg,assertion));
```

Comment:
Is there a reason you are passing whole auditMsg object? As you just need auditMessageType.getActiveParticipant() in this method